report.tex

```diff
@@ -68,6 +68,15 @@ \subsection{Selection of \textit{Consumer goods and services} sector}
 
 According to the 2017 budget review \cite{budgetReview} released by the Department of the National Treasury South Africa, the \textit{Consumer goods and services} sector was the main contributor to economic growth and job creation in 2016. This is found not to be a recent trend as an economic output presentation by STANLIB in 2013 \cite{stanlibReport} claims that although the South African economy had recovered after the 2008 crash, the recovery was mostly consumer based rather than production based. Consequently the \textit{Consumer goods and services} sector had outperformed all the other sectors and accounted for $70\%$ of the economy. This leads to the observation that the \textit{Consumer goods and services} sector and has grown steadily over the past year and is likely to continue to do so for the next year as the Department of the National Treasury South Africa predicts an economic growth of $1.3\%$ in 2017 and $2\%$ in 2018. 
 
+\section{Future prospects of British American Tobacco(BAT)}
+BAT is responsible for the sale of $90~\%$ of the cigarettes sold to  around $6.5~million$ adult smokers in South Africa. BAT also possesses around 200 trademarks which has resulted in it being one of the biggest marketshares worldwide \cite{BAT_hist}. South Africa currently holds the third largest share of the profits generated by BAT worldwide with a percentage figure of $8~\%$ \cite{BAT_hist}. Since the majority of their profits are generated internationally, BAT has reduced risk on political and economic events that take place in South Africa.\\
```

Comment:
Please check spaces in section 'Future prospects of British American Tobacco(BAT)}'.